README.md

```diff
@@ -50,21 +41,58 @@ _How can I build and deploy it (lots of this will be links out as the steps will
 * CI provided by CircleCI: [concordances-rw-dynamodb](https://circleci.com/gh/Financial-Times/concordances-rw-dynamodb)
 
 ## Service endpoints
-_What are the endpoints offered by the service_
+See the api/api.yml for the swagger definitions of these endpoints
 
-e.g.
 ### GET
 
-Using curl:
-
-    curl http://localhost:8080/_<INSERT SEPCIFIC URL HERE>_| json_pp`
-
-_Explain what the response should represent_
-
-Based on the following [google doc](_<INSERT API DOCUMETATION HERE>_)
+Using curl:  
+  _request:_
+  
+     curl -X GET "https://user:pass@prod-up.ft.com/__concordances-rw-dynamodb/concordances/4f50b156-6c50-4693-b835-02f70d3f3bc0" -H  "accept: application/json; charset=UTF-8"
+   
+ _response:_  
+ 
+    HTTP/1.1 200 OK
+    Content-Type: application/json
+    X-Request-Id: transaction ID, e.g. tid_etmIWTJVeA
+
+    {
+      "uuid": "4f50b156-6c50-4693-b835-02f70d3f3bc0",
+      "concordedIds": ["7c4b3931-361f-4ea4-b694-75d1630d7746", "1e5c86f8-3f38-4b6b-97ce-f75489ac3113", "0e5033fe-d079-485c-a6a1-8158ad4f37ce"]
+    }
+ 
+  _request:_
+  
+    curl -X GET "https://user:pass@prod-up.ft.com/__concordances-rw-dynamodb/concordances/__count" -H  "accept: text/plain"
+    HTTP/1.1 200 OK
+    Content-Type: text/plain
+    100
+
+### PUT
+Using curl:  
+  _request:_
+```
+curl -X PUT "https://user:pass@prod-up.ft.com/__concordances-rw-dynamodb/concordances/4f50b156-6c50-4693-b835-02f70d3f3bc0" -H  "accept: application/json" -H  
+"content-type: application/json; charset=utf-8" -d 
+"{  
+    "uuid": "4f50b156-6c50-4693-b835-02f70d3f3bc0",  
+    "concordedIds": [
+       "7c4b3931-361f-4ea4-b694-75d1630d7746 ",
+       "1e5c86f8-3f38-4b6b-97ce-f75489ac3113",
+       "0e5033fe-d079-485c-a6a1-8158ad4f37ce"
+         ]
+ }"
+```
+### DELETE
+Using curl:  
+  _request:_
+  
+    curl -X DELETE "https://api.ft.com/__concordances-rw-dynamodb/concordances/4f50b156-6c50-4693-b835-02f70d3f3bc0" -H  "accept: application/json"
+
+Based on the following [google doc](https://docs.google.com/document/d/1SFm7NwULX0nGqzfoX5JQGWZcd918YBwEGuO10kULovQ/edit?ts=591d86df#)
 
 ## Utility endpoints
-_Endpoints that are there for support or testing, e.g read endpoints on the writers_
+N/A  
```

Comment:
All of endpoints on this service are strictly utility endpoints as normal service will be that messages are consumed by kafka and processed rather than calling individual endpoints.